components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -25,7 +25,8 @@ spec:
           imagePullPolicy: Always
```

Comment:
kept Always here because this seems to be used internally, not in releases, for testing with the :latest image-tag of odh-notebook-controller, which necessitates imagePullPolicy: Always